lib/streamlit/runtime/scriptrunner/magic.py

```diff
@@ -15,6 +15,13 @@
 import ast
 import sys
 
+from typing_extensions import Final
+
+# When a Streamlit app is magicified, we insert a `magic_funcs` import near the top of
+# its module's AST:
+# import streamlit.runtime.scriptrunner.magic_funcs as __streamlitmagic__
+MAGIC_MODULE_NAME: Final = "__streamlitmagic__"
```

Comment:
This import alias was formerly `__streamlit__` - I changed it to `__streamlitmagic__` for clarity.